galeracluster/source/glossary.rst

```diff
@@ -6,6 +6,13 @@
 .. glossary::
    :sorted:
 
+
+   Galera Arbitrator
+      If the expense of adding, for example, a third datacenter is too high, you can use the Galera arbitrator. An arbitrator is a member of the cluster which participates in voting, but not in actual replication.

```

Comment:
I do not think this should be framed as an expense-based solution.

Galera Arbitrator is a separate process that is used to break ties in clusters with an even number of nodes, so that a network outage can not result in the cluster splitting into two equal parts with neither having the majority of nodes.  An arbitrator is a member of the cluster which participates in voting, but not in actual replication.